### ISSUE TITLE
Lativ Theme - On Activation, header is missing from the Blog Home Template

### ISSUE THREAD

**Issue by masperber** [Automattic/themes#7313]
### Quick summary

On WordPress.com sites, after activating the Lativ theme, the Blog Home Template is missing the header and displays the message "Template part has been deleted or is unavailable: header"

If you edit the template, the Header can be removed, and the template part can be added again without issue.

### Steps to reproduce

1. Create a site on WordPress.com with a theme other than Lativ active.
2. Change the theme to Lativ.

I did not test whether this bug affects sites that are created with the Lativ theme.

### What you expected to happen

The Header should be displayed without an error message.

### What actually happened

The Blog Home Template displays the message "Template part has been deleted or is unavailable: header" in place of the header.

### Platform (Simple, Atomic, or both?)

Simple, Atomic

### Reproducibility

Consistent

### Severity

All

### Available workarounds?

Yes, easy to implement

### Workaround details

The workaround consists of removing the Template Part Block from the header group and then adding a new one, referencing the Header template part.

**Comment by masperber** [comment]
Customer report 6662306-zen

**Comment by mikachan** [comment]
This will hopefully be fixed by this GB PR: https://github.com/WordPress/gutenberg/pull/53423

**Comment by cuemarie** [comment]
📌 SCRUBBING : RESULT - *Replicated / Could Not Replicate / Uncertain*
- Tested on Simple – Replicated

📌 FINDINGS/SCREENSHOTS/VIDEO
- Replicated just as described!

#### Screenshots/Recordings

https://github.com/Automattic/themes/assets/27249804/aa75b7a6-a969-4761-b5e9-4b249795de9d

📌 ACTIONS
- Assigned to @Automattic/theam 
- Fix incoming from core, thanks for noting that @mikachan ! Leaving in Themes' project boards _just in case_ that GB PR does not resolve 👍

**Comment by katiebethbrown** [comment]
@mikachan - Should this be resolved now that #53423 is merged? I just ran into it spinning up a new site with the Sten theme.

Edit - Nevermind, I see that the merged PR is for the GB 16.5 milestone. I'm assuming that means we'll see this resolved once we upgrade wpcom to use that version. 👍

**Comment by cuemarie** [comment]
> I'm assuming that means we'll see this resolved once we upgrade wpcom to use that version.

I believe that's correct, @katiebethbrown !

**Comment by mikachan** [comment]
Yep that's right, the fix will be in Gutenberg 16.5 so this should be fixed across wpcom when that version is being used (around August 24-25).
